src/yoroi-wallets/utils/utils.ts

```diff
@@ -98,9 +116,31 @@ export const Quantities = {
   denominated: (quantity: Quantity, denomination: number) => {
     return Quantities.quotient(quantity, `${10 ** denomination}`)
   },
+  atomic: (data: Quantity | BigNumber | string | number, denomination: number) => {
```

Comment:
dont add any more dependencies on BigNumber

Reply:
depend on Quantity, and thats it
if its not a Quantity, whatever created it should update to return a Quantity

Reply:
also, what is this?

Reply:
renamed to `fixed`, I'd like to receive anything and transform it into a quantity, maybe I need to change the logic to use `asQuantity` and pass it here later, let me check that.